src/main/cluster-detectors/distribution-detector.ts

```diff
@@ -60,6 +60,18 @@ export class DistributionDetector extends BaseClusterDetector {
       return { value: "custom", accuracy: 10};
     }
 
+    if (await this.isOpenshift()) {
+      return { value: "openshift", accuracy: 90};
+    }
+
+    if (this.isCustom()) {
+      return { value: "custom", accuracy: 10};
+    }
+
+    if (await this.isOpenshift()) {
+      return { value: "openshift", accuracy: 90};
+    }
+
```

Comment:
This is very weird. Why are we detecting twice?

Reply:
Weird indeed, it probably came from rebase. Will fix.